docs/setup-and-usage.md

```diff
@@ -109,112 +109,114 @@ cp configs/.env.example configs/.env # Edit the file after the copy.
 
-{% hint style="info" %}
-tested with [google cloud](https://cloud.google.com), but should work with any k8s cluster.
-{% endhint %}
+A Helm chart for installing telliot on Kubernetes
 
-* Install [`gcloud`](https://cloud.google.com/sdk/docs/install)
-* Install [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl)
-* Create a k8s cluster with a single node
-* Login to the cluster
+[A guide for installing helm can be found here](https://helm.sh/docs/intro/install/)
 
-```bash
-gcloud auth login --project projectName
-gcloud container clusters get-credentials main --zone europe-west2-a --project projectName
-```
+## telliot Configuration
 
-* Deploy the `cli` \(by default deployed to run as a miner\)
+Include telliot configuration files in the files directory of this chart.
 
-```bash
-git clone https://github.com/tellor-io/telliot
-cd telliot
-export INSTANCE_NAME=lat # Use max 3 characters due to k8s limitation for port names.
-export DEPL_NAME=telliot-m # This is the name of the deployment file.
-export CFG_FOLDER=.local/configs/$DEPL_NAME-$INSTANCE_NAME # Configs will be copied to this folder.
-export DEPL_INSTANCE_NAME=$DEPL_NAME-$INSTANCE_NAME
-mkdir -p $CFG_FOLDER
+These files should be:
 
-# Create the secret file.
-cp configs/.env.example $CFG_FOLDER/.env # Edit the file after the copy.
+- .env
+- index.json
+- manualData.json
 
-touch $CFG_FOLDER/config.json # Create an empty file and if needed overwrite the defaults.
+Run the following to use the default configuration files:
 
-# Copy the manual data file.
-cp configs/manualData.json $CFG_FOLDER/manualData.json
+```bash
+cp configs/index.json configs/manualData.json configs/helm/files/
+cp configs/.env.example configs/helm/files/.env # you will need to edit this file with your own secrets after copying
+```
+
+Optionally you can also include config.json if you would like to override any default config values.
 
-# Apply the configs.
-kubectl create secret generic $DEPL_INSTANCE_NAME --from-env-file=$CFG_FOLDER/.env
-kubectl create configmap $DEPL_INSTANCE_NAME \
-  --from-file=configs/index.json \
-  --from-file=$CFG_FOLDER/config.json \
-  --from-file=$CFG_FOLDER/manualData.json \
-  -o yaml --dry-run=client | kubectl apply -f -
+If you would like to use separate .env or config.json files for the mining and dataserver instances, copy those files under their respective directory in config/helm/files.
+For example to use a unique config.json and .env for mining and dataserver instances run:
 
-# Copy the manifest and run it.
-cp configs/manifests/$DEPL_NAME.yml $CFG_FOLDER/$DEPL_NAME.yml
-sed -i "s/$DEPL_NAME/$DEPL_INSTANCE_NAME/g" $CFG_FOLDER/$DEPL_NAME.yml
-kubectl apply -f $CFG_FOLDER/$DEPL_NAME.yml
+```bash
+# Mining instance
+cp configs/config.json configs/helm/files/mine/
+cp configs/.env.example configs/helm/files/mine/.env
+# dataserver instance
+cp configs/config.json configs/helm/files/dataserver/
+cp configs/.env.example configs/helm/files/dataserver/.env
```

Comment:
instead of this, I was thinking somehting like what we had before where the config folder is not hard coded, but used from the instance and deployment name like
```
export INSTANCE_NAME=lat
export DEPL_NAME=telliot-m 
export CFG_FOLDER=configs/helm/files/$DEPL_NAME-$INSTANCE_NAME # Configs will be copied to this folder.
```

Reply:
my thought here is variablizing the folder name will be more confusing than just using the hard-coded folder names. This will also need to be pulled into the helm file getters. I'd prefer to just leave this as is.